### PR TITLE
Fix doc and rotation curves

### PR DESCRIPTION
**Pull request by skhiat**
Fix doc and rotation for curve distribution mission on PR various_fixes_3.

### CODE REVIEW

**Review by sebastienlagarde — APPROVED** [review]

File: com.unity.render-pipelines.core/Documentation~/Common/lens-flare-data-driven-asset.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ The Lens Flare Element asset has the following properties:
 | ---------------- | ------------------------------------------------------------ |
 | Colors           | The range of colors that this asset applies to the lens flares. You can use the **Position Spacing** curve to determine how this range affects each lens flare. |
 | Position Variation | Adjust this curve to change the placement of the lens flare elements in the **Lens Spread**. |
-| Rotation | The uniform angle of rotation (in degrees) applied to each element distributed along the curve. |
+| Rotation | The uniform angle of rotation (remap from -180° to 180°) applied to each element distributed along the curve. |
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.core/Documentation~/Common/lens-flare-data-driven-asset.md:151]: Changed for clarity:

| Rotation | The uniform angle of rotation (in degrees) applied to each element distributed along the curve. </br>This value ranges from -180° to 180°. |

File: com.unity.render-pipelines.high-definition/Documentation~/lens-flare-data-driven-asset.md [hunk 2]
```diff
@@ -148,7 +148,7 @@ The Lens Flare Element asset has the following properties:
 | ---------------- | ------------------------------------------------------------ |
 | Colors           | The range of colors that this asset applies to the lens flares. You can use the **Position Spacing** curve to determine how this range affects each lens flare. |
 | Position Variation | Adjust this curve to change the placement of the lens flare elements in the **Lens Spread**. |
-| Rotation | The uniform angle of rotation (in degrees) applied to each element distributed along the curve. |
+| Rotation | The uniform angle of rotation (remap from -180° to 180°) applied to each element distributed along the curve. |
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/lens-flare-data-driven-asset.md:151]: Same again: 

| Rotation | The uniform angle of rotation (in degrees) applied to each element distributed along the curve. </br>This value ranges from -180° to 180°. |

File: com.unity.render-pipelines.universal/Documentation~/lens-flare-data-driven-asset.md [hunk 3]
```diff
@@ -148,7 +148,7 @@ The Lens Flare Element asset has the following properties:
 | ---------------- | ------------------------------------------------------------ |
 | Colors           | The range of colors that this asset applies to the lens flares. You can use the **Position Spacing** curve to determine how this range affects each lens flare. |
 | Position Variation | Adjust this curve to change the placement of the lens flare elements in the **Lens Spread**. |
-| Rotation | The uniform angle of rotation (in degrees) applied to each element distributed along the curve. |
+| Rotation | The uniform angle of rotation (remap from -180° to 180°) applied to each element distributed along the curve. |
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.universal/Documentation~/lens-flare-data-driven-asset.md:151]: And again:

| Rotation | The uniform angle of rotation (in degrees) applied to each element distributed along the curve. </br>This value ranges from -180° to 180°. |

**Review by Vic-Cooper — CHANGES_REQUESTED** [review]
Re-structured the description for clarity.

**Review by erikabar — APPROVED** [review]

**Review by sebastienlagarde — APPROVED** [review]

**Comment by sebastienlagarde** [comment]
Change are minimanl. Doc correctoin done. Merging

**Pull request merged**